src/sentry/api/endpoints/project_details.py

```diff
@@ -408,6 +436,10 @@ def put(self, request, project):
             if project.update_option('sentry:relay_pii_config', result['relayPiiConfig']):
                 changed_proj_settings['sentry:relay_pii_config'] = result['relayPiiConfig'].strip(
                 ) or None
+        if result.get('symbolSources') is not None:
+            if project.update_option('sentry:symbol_sources', result['symbolSources']):
+                changed_proj_settings['sentry:symbol_sources'] = result['symbolSources'] \
+                    .strip() or None
```

Comment:
When would this hit the `or None` case and be a valid source payload? Does the vaildator allow empty spaces?

Reply:
The code has changed since, but `or None` is still being hit when someone clears the input field. FWIW, the input field is only a temporary thing and will be replaced with more sophisticated UI.